src/app/auth/views/Auth/SignupBlog.tsx

```diff
@@ -26,10 +27,28 @@ const SignupAuth = () => {
 
 export default function SignupBlog(): JSX.Element {
   return (
-    <div className="flex flex-col items-center justify-center py-3 px-5 lg:py-16 lg:px-40">
-      <div className="flex h-full w-full flex-col items-center justify-center space-y-7 px-5 lg:flex-row lg:space-y-0 lg:space-x-48">
-        <div className="flex w-full max-w-md flex-col justify-between space-y-3 lg:space-y-10">
-          <SignupAuth />
+    <div className="hidden flex-col items-center justify-center overflow-hidden py-3 xs:flex lg:py-10">
+      <div className="flex w-full flex-row bg-gradient-to-br from-blue-20 to-white">
+        <div className="mt-5 mb-10 ml-11 flex w-full max-w-xs flex-col items-start justify-center">
+          <p className="text-2xl font-semibold">
+            Keep those now clean files safe, <span className="text-primary">join Internxt for free</span>
```

Comment:
This text only has to be in English because the registration form will only be used somewhere on the blog or website, right?

Reply:
This signup form is only used for blog posts.